.env.example

```
@@ -23,17 +24,12 @@ NEXT_PUBLIC_POSTHOG_HOST=https://app.posthog.com
 NEXT_PUBLIC_FORMBRICKS_API_HOST=https://app.formbricks.com
 
-# Security
-NEXT_PUBLIC_HMAC_SECRET_KEY=your_hmac_secret_key
-
 # Integration URLs
 NEXT_PUBLIC_SLACK_SERVER=https://slack.potpie.ai
-NEXT_PUBLIC_SENTRY_CLIENT_ID=your_sentry_client_id
-NEXT_PUBLIC_SENTRY_CLIENT_SECRET=your_sentry_client_secret
 NEXT_PUBLIC_SENTRY_REDIRECT_URI=https://your-app-domain.com/integrations/sentry
 NEXT_PUBLIC_LINEAR_REDIRECT_URI=https://your-app-domain.com/integrations/linear
+NEXT_PUBLIC_JIRA_REDIRECT_URI=https://your-app-domain.com/integrations/jira
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_JIRA_REDIRECT_URI key should go before the NEXT_PUBLIC_LINEAR_REDIRECT_URI key

(UnorderedKey)